src/hooks/usePolling.ts

```diff
@@ -2,17 +2,18 @@ import { useState } from "react";
 import { ApolloQueryResult, OperationVariables } from "@apollo/client";
 import Cookies from "js-cookie";
 import { DISABLE_QUERY_POLLING } from "constants/cookies";
-import { pollInterval } from "constants/index";
+import { FASTER_POLL_INTERVAL, DEFAULT_POLL_INTERVAL } from "constants/index";
 import { useNetworkStatus } from "./useNetworkStatus";
 import { usePageVisibility } from "./usePageVisibility";
 
 type usePollingType = {
   (
-    startPolling: (pollInterval?: number) => void,
+    startPolling: (DEFAULT_POLL_INTERVAL?: number) => void,
     stopPolling: () => void,
     refetch?: (
       variables?: Partial<OperationVariables>
     ) => Promise<ApolloQueryResult<any>> | void,
+    shouldPollFaster?: boolean,
     initialPollingState?: boolean
```

Comment:
Since we are now in the 3 optional variable territory i think a refactor to use an object is in place. (Arguably we should have done it for 2 optional variables but 🤷 )